conda_smithy/templates/run_osx_build.sh.tmpl

```diff
@@ -93,17 +101,27 @@ if [[ "${BUILD_WITH_CONDA_DEBUG:-0}" == 1 ]]; then
 
     # Drop into an interactive shell
     /bin/bash
+  {%- else %}
+    echo "rattler-build don't support debug mode right now"
+  {%- endif %}
 else
 {% if test in ["native", "native_and_emulated"] %}
     if [[ "${HOST_PLATFORM}" != "${BUILD_PLATFORM}" ]]; then
         EXTRA_CB_OPTIONS="${EXTRA_CB_OPTIONS:-} --no-test"
     fi
 {% endif %}
+    {%- if conda_build_tool != "rattler-build" %}
     {{ BUILD_CMD }} ./{{ recipe_dir }} -m ./.ci_support/${CONFIG}.yaml \
         --suppress-variables ${EXTRA_CB_OPTIONS:-} \
         --clobber-file ./.ci_support/clobber_${CONFIG}.yaml \
         --extra-meta flow_run_id="$flow_run_id" remote_url="$remote_url" sha="$sha"
 
+    {%- else %}
+
+    {{ BUILD_CMD }} --recipe ./{{ recipe_dir }} -m ./.ci_support/${CONFIG}.yaml --output-dir ${MINIFORGE_HOME}/conda-bld
```

Comment:
`${EXTRA_CB_OPTIONS:-}` is not used here

Reply:
right now `rattler-build` has a limited support of options ( for example, we don't have `--no-test` that is used in `EXTRA_CB_OPTIONS` ).

That's why I didn't used it here

Reply:
some of the options that are used in `EXTRA_CB_OPTIONS` are not present yet in rattler-build 
( like `--output-id` ).

That's why I didn't used it

Reply:
`--output-id` is added in debug mode which is correctly handled right now. `--no-test` is not supported?

Reply:
you are right ! I've added EXTRA_CB_OPTIONS to rattler-build